templates/Program/programComments.html.twig

```diff
@@ -71,9 +83,19 @@
               </a>
             {% endif %}
           </div>
-          <div class="comment-text">
+          <div id="comment-text-wrapper-{{ comment.id }}" class="comment-text">
             <p id="comment-text-{{ comment.id }}">{{ comment.text }}</p>
           </div>
+          <div id="comment-translation-wrapper-{{ comment.id }}" class="comment-translation-wrapper" style="display: none">
+            <div id="comment-translation-credit-wrapper-{{ comment.id }}">
+              <span id="comment-translation-before-languages-{{ comment.id }}" class="comment-translation-credit"></span>
+              <span id="comment-translation-first-language-{{ comment.id }}" class="comment-translation-credit"></span>
+              <span id="comment-translation-between-languages-{{ comment.id }}" class="comment-translation-credit"></span>
+              <span id="comment-translation-second-language-{{ comment.id }}" class="comment-translation-credit"></span>
+              <span id="comment-translation-after-languages-{{ comment.id }}" class="comment-translation-credit"></span>
+            </div>
+            <p id="comment-text-translation-{{ comment.id }}" class="comment-text-translation" lang=""></p>
```

Comment:
do not use abbreviation `lang`

Reply:
I think the lang property in HTML is standard practice when setting a language of a component. When I look it up I am getting stuff like this https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang